src/reference/config/writing-platform-configs.rst

```diff
@@ -287,7 +287,11 @@ locations. The entire :term:`run directory` can be symlinked, as well as
 certain sub-directories.
 
 * ``run`` - the :term:`run directory` itself
-* ``log``
+* ``log`` (see :term:`workflow log directory`)
+* ``log/job`` The ``log/job`` directory contains more files than ``log``
+  and the risks (to the workflow) from losing them are smaller: Tasks
+  without log files might need re-running: Loss of workflow logs might
+  require re-running part or all of a workflow.
```

Comment:
> ``log/job`` directory contains more files than ``log``

the job/log dir is contained within log

> Tasks  without log files might need re-running

Data is generally created in work or share not log so loss of logs is not usually a concern.



---

I think we can strip the extra info, if we are going to add this sort of content, we should do it for all the dirs.


```suggestion
  * ``log/job`` - contains job scripts and outputs.
```